integration-tests/client/chainlink_models.go

```diff
@@ -989,13 +988,13 @@ observationSource                      = """
 """
 [pluginConfig]
 {{range $key, $value := .PluginConfig}}
-{{$key}} = {{$value}}
+{{$key}} = "{{$value}}"
```

Comment:
This will cause issues for the integer config values. I would recommend that you set the values from corresponding job spec like this -
"key":   fmt.Sprintf("\"%s\"", value),

Reply:
Oh good call, I forgot about possible ints